src/uct/api/uct.h

```diff
@@ -346,6 +346,34 @@ typedef enum uct_atomic_op {
 } uct_atomic_op_t;
 
 
+/**
+ * @brief All existing UCT operations
+ *
+ * This enumeration defines all available UCT operations.
+ */
+typedef enum uct_op {
```

Comment:
Some enum which has all ops in a single type was needed. First I considered using `uct_atomic_op_t` together with `UCT_IFACE_FLAG_....` but that didn't seem feasible